addons/web/static/src/views/relational_model.js

```diff
@@ -21,7 +21,14 @@ import { session } from "@web/session";
 import { FormArchParser } from "@web/views/form/form_arch_parser";
 import { ListConfirmationDialog } from "@web/views/list/list_confirmation_dialog";
 import { Model } from "@web/views/model";
-import { archParseBoolean, evalDomain, isNumeric, isRelational, isX2Many } from "@web/views/utils";
+import {
```

Comment:
Could be a [REF] instead of a [MOV]. I prefer to keep MOV for larger moves like an addon from enterprise to community, or an addon extracted from another, or the other way around